app/views/incomes/estimatedPayLandingPage.scala.html

```diff
@@ -0,0 +1,64 @@
+@*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@(employerName: String, empId: Int)(implicit request: Request[_],
+messages: Messages, user: controllers.auth.TaiUser, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer, partialRetriever: uk.gov.hmrc.play.partials.FormPartialRetriever)
+
+@import uk.gov.hmrc.play.views.html.helpers._
+@import uk.gov.hmrc.urls.Link
+@import includes.ptaHeader
+@import includes.cancelLink
+
+@header = @{
+    ptaHeader(
+        displayBackLink = true,
+        mainHeadingText = messages("tai.howToUpdate.preHeading", employerName),
+        preHeadingText = messages("tai.howToUpdate.preHeading", employerName),
+        headerWidthClasses = Some("column-full")
+    )
+}
+
+@main(
+title = messages("tai.howToUpdate.preHeading", employerName),
+user = Some(user),
+pageHeader = Some(header),
+articleClasses = None,
+gaCustomTitle = None
+){
+    <div>
+        <p>@messages("tai.incomes.landing.intro")</p>
+        <h2>@messages("tai.incomes.landing.subheading", employerName)</h2>
+    </div>
+    <p>
+        @Html(messages(
+                "tai.incomes.landing.you.can",
+                Link.toInternalPage(
+                    url=controllers.employments.routes.EndEmploymentController.employmentUpdateRemove(empId).toString,
+                    value=Some(messages("tai.incomes.landing.employment.ended.link", employerName)),
```

Comment:
From slack
```
Play i18n messages do not work well with HTML, unfortunately.  You would have to surround the `@messages(...)` with `@Html(...)`, but that then means the employer name is vulnerable to XSS (potentially).  So it's not ideal at all.
```

Reply:
As discussed, this has been used elsewhere in TAI and presumably the Pen testing would have picked it up if it was an issue. Is it ok to leave this as is?